### DISCUSSION TITLE
There are too many automatically generated versions

### DISCUSSION THREAD

**Discussion by oscxc** [verdaccio/verdaccio#3089] [Q&A]
"_distfiles": {
		"test-0.0.2.tgz": {
			"url": "https://registry.npmjs.org/test/-/test-0.0.2.tgz",
			"sha": "67d0610319d4308a39d1b25b0a074e709c4dc0be",
			"registry": "npmjs"
		},
		"test-0.0.3.tgz": {
			"url": "https://registry.npmjs.org/test/-/test-0.0.3.tgz",
			"sha": "87bd204807f9972cd9096202a40cd7eed4ead5e0",
			"registry": "npmjs"
		},
		"test-0.0.4.tgz": {
			"url": "https://registry.npmjs.org/test/-/test-0.0.4.tgz",
			"sha": "dc6536fe47a9437a0f3c586c0b3c05659ed5be9a",
			"registry": "npmjs"
		},
		"test-0.0.5.tgz": {
			"url": "https://registry.npmjs.org/test/-/test-0.0.5.tgz",
			"sha": "0565c62de9042d404c7af948884ca44e277a8f4c",
			"registry": "npmjs"
		},
		"test-0.0.6.tgz": {
			"url": "https://registry.npmjs.org/test/-/test-0.0.6.tgz",
			"sha": "f74c095e789568e9c8a004602d695d85cf8f2334",
			"registry": "npmjs"
		},
How do I avoid making the wrong versions?

**Comment by juanpicado** [comment]
???? Could you elaborate?
